app/views/users/_user.html.erb

```diff
@@ -50,6 +50,7 @@
       </div>
 
       <div class="col">
+      <% if current_user == user %>
```

Comment:
Nice! Could you think of a way to refactor this using authorize and the policy helper methods?

Reply:
Agreed